Libs/MRML/Widgets/qMRMLTableView.cxx

```diff
@@ -219,14 +224,16 @@ vtkMRMLTableNode* qMRMLTableView::mrmlTableNode()const
 //------------------------------------------------------------------------------
 bool qMRMLTableView::transposed()const
 {
-  VERIFY_TABLE_MODEL_AND_NODE_RETURN_VALUE(transposed, false);
+  Q_D(const qMRMLTableView);
+  CTK_CHECK_AND_RETURN_FALSE_IF_FAIL(verifyTableModelAndNode)
```

Comment:
Thanks Jc, these changes look good!

The only thing that I don't like is that d-> is hidden in the macro. I think that instead of

CTK_CHECK_AND_RETURN_FALSE_IF_FAIL(verifyTableModelAndNode) 

this would be more clear:

CTK_CHECK_AND_RETURN_FALSE_IF_FAIL(d->verifyTableModelAndNode)

For example, without d-> it's not obvious why you need the Q_D macro.